spokestack/vad/webrtc.py

```diff
@@ -0,0 +1,74 @@
+"""
+This module contains the webrtc component for
+voice activity detection (vad)
+"""
+
+import webrtcvad  # type: ignore
+
+from spokestack.context import SpeechContext
+
+
+MODE = {"quality": 0, "low_bitrate": 1, "aggresive": 2, "very_aggresive": 3}
```

Comment:
this would be simpler if you created named constants QUALITY, LOW_BITRATE, AGGRESSIVE, VERY_AGGRESSIVE
then the client could pass symbolic constants instead of strings and you wouldn't need to map them to webrtc's values